### PR TITLE
Updated the vSphere readme for the new make config

### PR DESCRIPTION
**Pull request by jjasghar**
- Fixed the ordering of the questions
- Fixed some extra whitespace

Signed-off-by: JJ Asghar <jj@chef.io>

### CODE REVIEW

**Comment by jjasghar** [comment]
CLA signed, please recheck.

File: phase1/vsphere/README.md [hunk 1]
```diff
@@ -120,15 +120,15 @@ number of nodes (phase1.num_nodes) [4] (NEW) 4
...
+* phase1.ssh_user - This field is not used for vSphere Deployment. Leave it blank and hit enter.
 ```
-SSH user to login to OS for provisioning (phase1.ssh_user) [] (NEW) 
+SSH user to login to OS for provisioning (phase1.ssh_user) [] (NEW)
 ```
 
 * Select the provider, in this case it would be vsphere.
 ```
-cloud provider: gce or vsphere (phase1.cloud_provider) [vsphere] (NEW) vsphere
+cloud provider: gce, azure or vsphere (phase1.cloud_provider) [vsphere] (NEW) vsphere
```
Comment (abrarshivani) [inline phase1/vsphere/README.md]: @jjasghar Currently, there is no support for azure cloudprovider in k8s-anywhere. Can you please change this?

File: phase1/vsphere/README.md [hunk 2]
```diff
@@ -223,22 +223,25 @@ For Kubernetes release 1.6 and above use `docker.io/cnastorage/k8s-ignition:v2`.
...
 ```
-kubernetes version (phase2.kubernetes_version) [v1.6.5] (NEW) v1.6.5
+installer container (phase2.installer_container) [docker.io/cnastorage/k8s-ignition:v2] (NEW) docker.io/cnastorage/k8s-ignition:v2
 ```
 
-* Set bootstrap provider to ignition
+* Registry to be used by Kubernetes
 ```
-bootstrap provider (phase2.provider) [ignition] (NEW) ignition
+docker registry (phase2.docker_registry) [gcr.io/google-containers] (NEW)
```
Comment (abrarshivani) [inline phase1/vsphere/README.md]: Currently, this is before installer container. Can you please change this?

File: phase1/vsphere/README.md (hunk 1 above)
Comment (jjasghar) [reply phase1/vsphere/README.md]: I copy pasted this directly from the install script, is that wrong?

File: phase1/vsphere/README.md (hunk 2 above)
Comment (jjasghar) [reply phase1/vsphere/README.md]: I copy this order this directly from the install script, is that wrong?

File: phase1/vsphere/README.md (hunk 1 above)
Comment (abrarshivani) [reply phase1/vsphere/README.md]: @jjasghar You are right. My comment was for ```make docker-dev```. Yet, this is for the container ```docker run -it -v /tmp:/tmp --rm --env="PS1=[container]:\w> " --net=host cnastorage/kubernetes-anywhere:latest /bin/bash``` which is mentioned in README

File: phase1/vsphere/README.md (hunk 2 above)
Comment (abrarshivani) [reply phase1/vsphere/README.md]: ditto

**Pull request merged**